examples/vulkan/Vulkan.cpp

```diff
@@ -470,11 +470,11 @@ class VulkanExample
             // -- VK_LAYER_GOOGLE_unique_objects
             // These layers perform error checking and warn about bad or sub-optimal Vulkan API usage
             // VK_LAYER_LUNARG_monitor appends an FPS counter to the window title
-            if (!std::strcmp(layer.layerName, "VK_LAYER_LUNARG_standard_validation"))
+            if (std::string_view(layer.layerName) == std::string_view("VK_LAYER_LUNARG_standard_validation"))
```

Comment:
```suggestion
            if (std::string_view(layer.layerName) == "VK_LAYER_LUNARG_standard_validation")
```
We can compare string views to `const char*`s whether they're Vulkan constant or string literals so we don't need to explicitly construct a second string view. This applies to many other places in this PR so I'll only mention it once.

Reply:
You're right, I completely overlook string literals in the process. Commit 2767be2e9e7961f7a4bc6032be9d957f6ed6c20c should fix those instances.